example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -84,8 +85,13 @@ describe('The Ingest Granule failure workflow', () => {
         },
       ];
 
-      // delete the granule record from DynamoDB if exists
-      await granuleModel.delete({ granuleId: inputPayload.granules[0].granuleId });
+      await deleteGranule({ prefix: config.stackName, granuleId: inputPayload.granules[0].granuleId });
```

Comment:
I'll circle back to this, but it's not immediately clear why granules[0] is explicitly called before the map on the granules object.

Reply:
Hm that is a great point. I will remove the map since it's not needed but I am not sure why we need to delete the granule record if we are making it fail by adding a non existent file to it.

Reply:
[`815243d` (#2254)](https://github.com/nasa/cumulus/pull/2254/commits/815243d4a145724b531e1380b1140e63f738bb96)

Reply:
👀    Hmm, you're right.   This test is validating a failure, so pre-cleanup isn't *really* required, particularly with the other changes to granule cleanup we're making. 

Interestingly enough, we're *indexing* in the cleanup which isn't great if we changed that fixture, but I'm not incredibly concerned re: that for now.